starter/js/atm.js

```diff
@@ -1,23 +1,72 @@
 //Begin with the document ready function
+$(document).ready(function() {
+  //Checking account deposit function
+  //On click of the depositChecking button
 
-    //Checking account deposit function
+  $("#depositChecking").click(function(event) {
+      console.log(event);
 
-      //On click of the depositChecking button
+      //Get value from the amountChecking input field
+      var depositAmount = parseInt($("#amountChecking").val(), 10);
+      var currentBalance = parseInt($('#checkingBalance').text().replace('$', ''), 10);
+      //Take that value and add it to the existing value in the checkingBalance div
+      var totalBalance = depositAmount + currentBalance;
+      $("#checkingBalance").text("$" + totalBalance);
+  });
 
-        //Get value from the amountChecking input field
 
-        //Take that value and add it to the existing value in the checkingBalance div
+  //Checking account withdrawl funtion
+  //On click of the withdrawChecking button
+  //Get value from the amountChecking input field
 
-    //Checking account withdrawl funtion
+    $("#withdrawChecking").click(function(event) {
+      console.log(event);
 
-      //On click of the withdrawChecking button
+      //Get value from the amountChecking input field
+      var withAmount = parseInt($("#amountChecking").val(), 10);
+      var currentBalance = parseInt($('#checkingBalance').text().replace('$', ''), 10);
+      //Take that value and add it to the existing value in the checkingBalance div
 
-        //Get value from the amountChecking input field
+      var totalBalance = currentBalance - withAmount;
+
+      if (totalBalance >= withAmount) {
+        $("#checkingBalance").text("$" + totalBalance);
+      }
+  });
+
+    $("#depositSavings").click(function(event) {
```

Comment:
If you had wanted to extract the logic for both the checking and savings features into one block of code instead of copying and pasting for both, how could you have done it? The secret to DRYing up code with functions is to identify which parts of the code are the same, and which parts are different - the parts that are the same you can keep identical but move into a function, the parts that are different become arguments to that function. You replace the parts that are specific to each implementation (in this case, the names of the selectors) with arguments so they can be abstracted to any situation.